.github/workflows/build_web.yaml

```diff
@@ -3,6 +3,11 @@ name: Build web
 on:
   workflow_call:
 
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: glaciation-heu/kg_exporter
+  SHORT_IMAGE_NAME: kg_exporter
```

Comment:
It would be better to remove the unused value SHORT_IMAGE_NAME

Reply:
done